repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/atomfeed/AuthFeedReader.java

```diff
@@ -1,7 +1,8 @@
 package org.openrepose.filters.clientauth.atomfeed;
 
+import org.openrepose.services.serviceclient.akka.AkkaServiceClientException;
+
 public interface AuthFeedReader {
-   
-   CacheKeys getCacheKeys();
-   String getFeedId();
+
+   CacheKeys getCacheKeys() throws AkkaServiceClientException;
```

Comment:
Also this is another case where if you need to pass out an exception it should be a localized one. I'm pretty sure that anyone using something of this type of interface isn't going to care that the implementer underneath is using akka.